pallets/settlement/Cargo.toml

```diff
@@ -22,30 +22,33 @@ sp-serializer = { git = "https://github.com/paritytech/substrate", default-featu
 sp-runtime = { git = "https://github.com/paritytech/substrate", default-features = false, tag = "v2.0.0" }
 sp-version = { git = "https://github.com/paritytech/substrate", default-features = false, tag = "v2.0.0" }
 sp-api = { git = "https://github.com/paritytech/substrate", default-features = false, tag = "v2.0.0" }
-pallet-timestamp = { git = "https://github.com/paritytech/substrate", default-features = false, tag = "v2.0.0" }
 
 frame-system = { git = "https://github.com/paritytech/substrate", default-features = false, tag = "v2.0.0" }
 frame-support = { git = "https://github.com/paritytech/substrate", default-features = false, tag = "v2.0.0" }
+pallet-scheduler = { git = "https://github.com/paritytech/substrate", default-features = false, tag = "v2.0.0" }
```

Comment:
I think we really don't need this. As we are using the `Anon` trait to interact with the pallet_scheduler.